### ISSUE TITLE
Issue running Weathermapper

### ISSUE THREAD

**Issue by mattie47** [pblasquez/weathermapper#8]
Just a note in case anyone else comes across this:

After cloning the repo, and modifying the config files, I was getting the following problem:

```
librenms@librenmsdev:/opt/weathermapper/conf.d$ php ../weathermapper.php 
PHP Warning:  require_once(../includes/weathermapper.inc.php): failed to open stream: No such file or directory in /opt/weathermapper/weathermapper.conf.php on line 41
PHP Fatal error:  require_once(): Failed opening required '../includes/weathermapper.inc.php' (include_path='.:/usr/share/php') in /opt/weathermapper/weathermapper.conf.php on line 41
```

I got around this by modifying line 41 in weathermapper.conf.php:
```diff
// Weathermapper base directory
-$weathermapper_dir = dirname($argv[0]); 
+$weathermapper_dir = '/opt/weathermapper';
```

Thanks,

Matt

**Comment by laf** [comment]
Probably should be `$weathermapper_dir = __DIR__;`

**Comment by pblasquez** [comment]
I have merged PR #16 to fix this. Thank you.
